index.html

```diff
@@ -263,7 +264,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementaion guidance and best practices for working with VCs</li>
+                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
+                <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
+              </ul>
+            </li>
```

Comment:
Please delete all the text being added above.  We should not be listing any deliverables about protocols or APIs, whether normative or not.  Non-normative protocol and API deliverables will take just as much working group energy as normative ones (possibly more!) and we should be focusing our energies on improving the data model.  These would be an unnecessary and unhelpful distraction.

Reply:
The VCWG has a solid track record of prioritizing normative deliverables over non-normative ones.

The VCWG also has [existing non-normative notes](https://www.w3.org/TR/vc-imp-guide/) that we need to update to stay relevant, including guidance and exemplary work related to protocols that we already know exist and are being used to exchange Verifiable Credentials.

Reply:
@selfissued we need to be allowed to discuss these issues in order to ensure interoperability, and specifically to address the problems with VC-JWT (like no defined way to obtain a public key).

Reply:
In the event that any members of the VCWG feel that the group's focus is not where is ought to be, the appropriate action is to contact the group chairs.
Speaking as a chair, my first priority for the group is to see the normative deliverables accomplished. Notes and other non-normative deliverables always come second.

Reply:
Having said that, Notes and other non-normative deliverables do not need to be included in the charter in order for the group to publish them, and it is possible that some mentioned in the charter may not be published.
The list of non-normative deliverables included in the charter is a declaration of intention. It is not binding, nor is it limiting.

Reply:
> @selfissued we need to be allowed to discuss these issues in order to ensure interoperability, and specifically to address the problems with VC-JWT (like no defined way to obtain a public key).

I absolutely support discussing the implications of APIs and protocols on the VC Data Model and the relationships between them.  However, there don't need to be non-normative chartered API or protocol deliverables for those discussions to occur.

Reply:
I think it's clear now that there is consensus that the WG should be free to *discuss* APIs and protocols.

So I think we should focus our efforts on the remaining point of contention, which is whether or not the charter explicitly calls out that the WG intends to publish a number of NOTEs related to APIs and protocols and their interactions with the data model (provided it has time).

I do not find the argument that the group may become distracted or run out of time because the charter lists this information compelling. The only requirement for the charter period is that it sufficiently accounts for the completion of the normative items, which will be prioritized over non-normative ones. Whether the group becomes distracted or runs out of time is not something that the expression of non-normative items controls. WGs are voluntary and how participants choose to spend their time is totally out of the control of anyone except those members (and perhaps their employers).

If a member *really* wants to do work on a topic, they will do that whether a document is to be published by the WG or not. Telling that member to work on it in another group won't change that fact, they'll just spend less time in the WG. Therefore, IMO, the disagreement here seems to boil down to whether the objectors are ok with the WG publishing NOTEs on various topics.

To that point, my understanding is that a significant number of members want the WG to have the leeway to publish such NOTEs (with some specific topics in mind). It's also my understanding that the WG will not publish such NOTEs unless there is consensus in the WG to do so.

Given this, any objectors to including these topics as non-normative publication targets in the charter should be able to instead have any concerns about those NOTEs expressed within them before consenting to publication. I also expect that all WG participants would act in good faith to ensure that any published NOTEs would make it clear what the WG's position is on the topics therein regardless.

As a future WG member myself, I would object to the publication of any NOTE that did not accurately reflect the WG's position until it was adjusted to account for it. I would not object to other WG members doing work that they want to do that is clearly related to Verifiable Credentials (in the VCWG) -- and publishing NOTEs about their findings.

Again, this is all predicated on my assumption that the WG will prioritize normative items over non-normative ones -- which I think is a fair view.